### PR TITLE
remove old dockerfile

### PR DESCRIPTION
**Pull request by acourdavault**
## Description

Our Old docker file is not maintained and does not start a dev server,
The alternative is much simpler:
`docker run --rm -v $(pwd):/usr/src/ -w /usr/src/ -p 3000:3000 node:10-alpine yarn up`

Replaces https://github.com/linode/manager/pull/6229

## Type of Change
- Non breaking change ('update', 'change')

### CODE REVIEW

**Comment by er0k** [comment]
Should probably remove the scripts in package.json that use this Dockerfile:

```
    "docker:test": "docker build -f Dockerfile . -t 'manager' && docker run -it cloud --rm -v $(pwd)/packages/manager/src:/home/node/app/packages/manager/src -v $(pwd)/packages/linode-js-sdk/src:/home/node/app/packages/linode-js-sdk/src manager test",
    "docker:local": "docker build -f Dockerfile . -t 'manager' -t 'dev' && docker run -it --rm -p 3000:3000 -v $(pwd)/packages/manager/src:/home/node/app/packages/manager/src -v $(pwd)/packages/linode-js-sdk/src:/home/node/app/packages/linode-js-sdk/src manager start:docker",

```

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ If your intention is to start a development server for all projects, you have a
 
 To start all projects run `yarn up`.
 
+You can also use Docker without building any image:
+`docker run --rm -v $(pwd):/usr/src/ -w /usr/src/ -p 3000:3000 node:10-alpine yarn up`
```
Comment (er0k) [inline GETTING_STARTED.md]: I'm not able to get that command to work. Am I doing something wrong?

```
17:49:43 ehierhager@potato:~/work/devenv/repos/manager  (clean-dockerfile)
$ docker run --rm -v $(pwd):/usr/src/ -w /usr/src/ -p 3000:3000 node:10-alpine yarn up
yarn run v1.21.1
$ yarn install:all && yarn build:sdk && yarn start:all
$ yarn install --frozen-lockfile
[1/4] Resolving packages...
[2/4] Fetching packages...
warning url-loader@1.1.2: Invalid bin field for "url-loader".
info There appears to be trouble with your network connection. Retrying...
warning sha.js@2.4.11: Invalid bin entry for "sha.js" (in "sha.js").
warning requirejs@2.3.6: Invalid bin entry for "r.js" (in "requirejs").
info fibers_node_v8@3.1.5: The engine "node" is incompatible with this module. Expected version ">=8.0.0 <10.0.0". Got "10.19.0"
info "fibers_node_v8@3.1.5" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@2.0.7: The platform "linux" is incompatible with this module.
info "fsevents@2.0.7" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning " > eslint-plugin-cypress@2.10.3" has unmet peer dependency "eslint@>= 3.2.1".
warning "workspace-aggregator-47e5779f-6a7f-4970-a4f3-f016280446a5 > linode-manager > @material-ui/lab@4.0.0-alpha.29" has incorrect peer dependency "@material-ui/core@^4.5.0".
warning "workspace-aggregator-47e5779f-6a7f-4970-a4f3-f016280446a5 > linode-manager > react-async-script-loader@0.3.0" has incorrect peer dependency "react@^15.5.4".
warning "workspace-aggregator-47e5779f-6a7f-4970-a4f3-f016280446a5 > linode-manager > react-ga@2.7.0" has unmet peer dependency "prop-types@^15.6.0".
warning "workspace-aggregator-47e5779f-6a7f-4970-a4f3-f016280446a5 > linode-manager > chai-string@1.5.0" has unmet peer dependency "chai@^4.1.2".
warning "workspace-aggregator-47e5779f-6a7f-4970-a4f3-f016280446a5 > linode-manager > cypress-axe@0.5.3" has incorrect peer dependency "cypress@^3.1.1".
warning "workspace-aggregator-47e5779f-6a7f-4970-a4f3-f016280446a5 > linode-manager > storybook-react-router@1.0.8" has unmet peer dependency "react-router@^4.0.0||^5.0.0".
warning "workspace-aggregator-47e5779f-6a7f-4970-a4f3-f016280446a5 > linode-manager > @storybook/addon-actions > @storybook/api@5.3.9" has unmet peer dependency "regenerator-runtime@*".
warning "workspace-aggregator-47e5779f-6a7f-4970-a4f3-f016280446a5 > linode-manager > @storybook/addon-actions > react-inspector > storybook-chromatic@2.2.2" has unmet peer dependency "@storybook/core@3.* || 4.* || 5.*".
info If you think this is a bug, please open a bug report with the information provided in "/usr/src/yarn-error.log".
error An unexpected error occurred: "ENOENT: no such file or directory, lstat '/usr/src/packages/manager/node_modules/@jest/core/build/plugins/'".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by acourdavault** [comment]
Working on:

- [ ] checking why @er0k has a bug
- [ ] updating `package.json`
- [ ] checking that `devenv` build is able to support the new solution before i merge this
- [ ] cleaning up scripts using this Dockerfile

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -23,6 +23,9 @@ If your intention is to start a development server for all projects, you have a
 
 To start all projects run `yarn up`.
 
+You can also use Docker without building any image:
```
[the remark this answers is not in the document]
Comment (acourdavault) [reply GETTING_STARTED.md]: Ok, adding that

File: GETTING_STARTED.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (acourdavault) [reply GETTING_STARTED.md]: Hello for this the subject is a bit more complex.
I Use a Node official image for build and watch mode/dev purposes.
Nginx for serve purposes.

I am contacting ritchie for more info

[the remark this answers is not in the document]
Comment (Jskobos) [reply GETTING_STARTED.md]: Just take off the `-alpine`, all the Node default images use Debian

**Pull request merged**